docs/core/diagnostics/dotnet-counters.md

```diff
@@ -9,12 +9,20 @@ ms.date: 02/26/2020
 
 ## Install dotnet-counters
 
+### Option 1 - Install as dotnet global tool:
+
 To install the latest release version of the `dotnet-counters` [NuGet package](https://www.nuget.org/packages/dotnet-counters), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
 
 ```dotnetcli
 dotnet tool install --global dotnet-counters
 ```
 
+### Option 2 - Direct download:
+
+- Win ([x86](https://aka.ms/dotnet-counters/win-x86) | [x64](https://aka.ms/dotnet-counters/win-x64) | [arm](https://aka.ms/dotnet-counters/win-arm) | [arm-x64](https://aka.ms/dotnet-counters/win-arm64))
+- macOS ([x64](https://aka.ms/dotnet-counters/osx-x64))
+- Linux ([x64](https://aka.ms/dotnet-counters/linux-x64) | [arm](https://aka.ms/dotnet-counters/linux-arm) | [arm64](https://aka.ms/dotnet-counters/linux-arm64) | [musl-x64](https://aka.ms/dotnet-counters/linux-musl-x64) | [musl-arm64](https://aka.ms/dotnet-counters/linux-musl-arm64))
```

Comment:
There are a few things I think we should address in the design of these pages.

01. The **Description** section's content should be moved to the intro of the article after the **APPLIES TO** info, and then the **Description** header removed.
01. I think the way the install area is designed, it doesn't look good on the [published site](https://review.docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-sos?branch=pr-en-us-21490). I think we should remove the extra headers, they draw attention away from the main section **Install**.

    Summary of changes:
    - Change header from `Install dotnet-counters` to simply `Install`.
    - Add intro text to the `Install` header to help SEO.
    - Use bullet points instead of H3 for the two types of install.
    - Add some introduction to the binary download.
    - Change download from bullets to table.

    ````suggestion
    ## Install

    There are two ways to download and install `dotnet-counters`:

    - **dotnet global tool:**

      To install the latest release version of the `dotnet-counters` [NuGet package](https://www.nuget.org/packages/dotnet-counters), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:

      ```dotnetcli
      dotnet tool install --global dotnet-counters
      ```

    - **Direct download:**

      Download the tool executable that matches your platform:

      | OS  | Platform |
      | --- | -------- |
      | Windows | [x86](https://aka.ms/dotnet-counters/win-x86) \| [x64](https://aka.ms/dotnet-counters/win-x64) \| [arm](https://aka.ms/dotnet-counters/win-arm) \| [arm-x64](https://aka.ms/dotnet-counters/win-arm64) |
      | macOS   | [x64](https://aka.ms/dotnet-counters/osx-x64) |
      | Linux   | [x64](https://aka.ms/dotnet-counters/linux-x64) \| [arm](https://aka.ms/dotnet-counters/linux-arm) \| [arm64](https://aka.ms/dotnet-counters/linux-arm64) \| [musl-x64](https://aka.ms/dotnet-counters/linux-musl-x64) \| [musl-arm64](https://aka.ms/dotnet-counters/linux-musl-arm64) |
    ````

    ##### Preview:

    ![image](https://user-images.githubusercontent.com/67293991/99297519-38516680-27fd-11eb-93cd-815f25d6dc42.png)